index.js

```diff
@@ -0,0 +1,53 @@
+var map = document.querySelector(".contacts-map");
+var modalmap = document.querySelector(".modal-map");
+var close = document.querySelector(".map-close");
+            
+var writeus = document.querySelector(".writeus-button");
+var modalform = document.querySelector(".modal-write");
+var fio = modalform.querySelector("[name=user-name]");
+var formclose = document.querySelector(".write-close");
+var form = modalform.querySelector("form");
+var email = modalform.querySelector("[name=user-email]");
+
+
+map.addEventListener("click", function (evt) {
+    evt.preventDefault();
+    modalmap.classList.add("map-show");
+});
+close.addEventListener("click", function (evt) {
+    evt.preventDefault();
+    modalmap.classList.remove("map-show");
+});
+window.addEventListener("keydown", function(evt) {
+    if (evt.keyCode === 27) {
+        if (modalmap.classList.contains("map-show")) {
+            evt.preventDefault();
```

Comment:
what for?) what's the default behavior of click on Escape?

Reply:
![image](https://user-images.githubusercontent.com/116226611/208072995-7f2bb956-89c2-4bd7-b423-60a1104bb67b.png)
I took this from the lecture. If we press Esc and modalMap is already closed then what will we close? I unredstand that in this case we can do without Contains.

Reply:
hmm, maybe here it is really reasonable 👍 I trust Academy)